### ISSUE TITLE
Cannot install WP_Mock

### ISSUE THREAD

**Issue by gravityrail** [10up/wp_mock#153]
**Describe the bug**
Missing mockery/mockery dependency with constraint ^1.0.

**To Reproduce**
`composer install` with wp_mock produces this:

```
$ composer install --prefer-dist --no-progress --no-suggest
Loading composer repositories with package information
Installing dependencies (including require-dev) from lock file
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - 10up/wp_mock 0.4.2 requires mockery/mockery ^1.0 -> no matching package found.
    - 10up/wp_mock 0.4.2 requires mockery/mockery ^1.0 -> no matching package found.
    - Installation request for 10up/wp_mock 0.4.2 -> satisfiable by 10up/wp_mock[0.4.2].

Potential causes:
 - A typo in the package name
 - The package is not available in a stable-enough version according to your minimum-stability setting
   see <https://getcomposer.org/doc/04-schema.md#minimum-stability> for more details.
 - It's a private package and you forgot to add a custom repository to find it

Read <https://getcomposer.org/doc/articles/troubleshooting.md> for further common problems.
```

Time to update your dependencies?

**Comment by kudinovfedor** [comment]
Hi @gravityrail
As I see you install dependencies from composer.lock
I try to reproduce your error when remove from composer.lock block about package mockery/mockery and getting almost the same error
`
composer install --prefer-dist --no-progress --no-suggest
You are using the deprecated option "--no-suggest". It has no effect and will break in Composer 3.
Installing dependencies from lock file (including require-dev)
Verifying lock file contents can be installed on current platform.
Your lock file does not contain a compatible set of packages. Please run composer update.

  Problem 1
    - 10up/wp_mock is locked to version 0.4.2 and an update of this package was not requested.
    - 10up/wp_mock 0.4.2 requires mockery/mockery ^1.0 -> could not be found in any version, there may be a typo in the package name.

Potential causes:
 - A typo in the package name
 - The package is not available in a stable-enough version according to your minimum-stability setting
   see <https://getcomposer.org/doc/04-schema.md#minimum-stability> for more details.
 - It's a private package and you forgot to add a custom repository to find it

Read <https://getcomposer.org/doc/articles/troubleshooting.md> for further common problems.
`

Package 10up/wp_mock has require dependency "mockery/mockery": "^1.0", this info you can view in
vendor/10up/wp_mock/composer.json

Please try remove composer.lock and then install dependencies again. 
I think you get an error because of the lock file.

**Comment by unfulvio-godaddy** [comment]
This might be fixed by changes introduced by #167 and #170
